cpp/src/pypowsybl.cpp

```diff
@@ -246,6 +246,14 @@ SeriesArray* createImporterParametersSeriesArray(const std::string& format) {
     return new SeriesArray(callJava<array*>(::createImporterParametersSeriesArray, (char*) format.data()));
 }
 
+main_attributes* getMainAttributes(const JavaHandle& network) {
```

Comment:
here we should return a `shared_ptr` that will call `freeMainAttributes` on destruction to free memory

I would prefer "network_metadata" than "main_attributes" to be consistent with other places and because "main" sounds like other data is less important